hedera-node/src/main/java/com/hedera/services/txns/TransitionRunner.java

```diff
@@ -9,9 +9,9 @@
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * 
+ *
  *      http://www.apache.org/licenses/LICENSE-2.0
- * 
+ *
```

Comment:
I noticed that we are adding small changes to the license. Lets update the license in our IDEs so that we are not introducing those subtle changes to the Hedera team